en/README.md

```diff
@@ -54,7 +54,8 @@ TypeScript/JavaScript | [mavgen](getting_started/generate_libraries.md#mavgen) |
 Lua     | [mavgen](getting_started/generate_libraries.md#mavgen) | Y | Y | | 
 Swift   | [mavgen](getting_started/generate_libraries.md#mavgen) | Y | | | 
 Clojure | [clj-mavlink](https://github.com/WickedShell/clj-mavlink) | Y | Y | Y | Clojure MAVLink Bindings.
-Go      | [gomavlib](https://github.com/gswly/gomavlib) | Y | Y | Y | 
+Go      | [gomavlib](https://github.com/gswly/gomavlib) | Y | Y | Y |
+Haskell | [HaskMavlink](https://github.com/SweeWarman/HaskMavlink)| N | Y | N |  
```

Comment:
Thanks for this.
The library doesn't support MAVLink 1? :-(

Reply:
Not yet at least. I started working on V2 because I needed that urgently for a project. But I’ll also be adding V1 soon.

Reply:
Thanks! I'll merge now. Much appreciated!